docs/book/src/introduction/sway_quickstart.md

```diff
@@ -173,10 +173,15 @@ This starts a Fuel node with a volatile database that will be cleared when shut
 
 To deploy `counter_contract` on your local Fuel node, open a new terminal tab and run the following command from the root of the `wallet_contract` directory:
 
+> **Note**
+> You will require the plugin [forc-wallet](https://github.com/FuelLabs/forc-wallet) in order to sign your transactions.
+
 ```sh
-forc deploy --unsigned
+forc deploy --url localhost:4000 --unsigned
```

Comment:
By default `forc-deploy` tries to deploy to `DEFAULT_NODE_URL`, where it is declared as `pub const DEFAULT_NODE_URL: &str = "http://127.0.0.1:4000";`
```suggestion
forc deploy --unsigned
```

With latest `fuel-core` and `forc`&`forc-client` offered with fuelup, I can both use `forc deploy --unsigned` and `forc deploy --url localhost:4000 --unsigned`.

So I think it would better to keep this as it is and mention `--url` path in the note below.